File: Problems/Leetcode/3001-4000/3201-3300/3238.cpp
#include <bits/stdc++.h>

using namespace std;

static constexpr int MOD = 1e9 + 7;

class Solution {
public:
    int winningPlayerCount(int n, vector<vector<int>>& pick) {
        vector vec(n, vector<int>(11));
        for (auto& v : pick) {
            vec[v[0]][v[1]]++;
        }

        int ans = 0;
        for (int i = 0; i < n; ++i) {
            int ok = 0;
            for (int j = 0; j < 11; ++j) {
                if (vec[i][j] > i) {
                    ok = 1;
                    break;
                }
            }
            ans += ok;
        }
        return ans;
    }
};

int main() {
    Solution sol;

    return 0;
}

File: Problems/Leetcode/3001-4000/3201-3300/3238.py
from collections import defaultdict
from typing import List


class Solution:
    def winningPlayerCount(self, n: int, pick: List[List[int]]) -> int:
        ls = [defaultdict(int) for _ in range(n)]
        for u, v in pick:
            ls[u][v] += 1

        return sum(any(ls[i][v] > i for v in ls[i]) for i in range(n))
